website/docs/utilities/popover-primitive/partials/code/component-api.md

```diff
@@ -0,0 +1,123 @@
+    Provides a modifiers that needs to be applied to the popover element. It accepts an `anchoredPositionOptions` object as "named" argument, with different keys corresponding to different options (see below). These options are forwarded to underlying modifier `hds-anchored-position`, and in turn are passed down to the [Floating UI](https://floating-ui.com) library.
+    <Doc::ComponentApi as |C|>
+      <C.Property @name="anchoredPositionOptions.placement" @type="enum" @values={{array "top" "top-start" "top-end" "right" "right-start" "right-end" "bottom" "bottom-start" "bottom-end" "left" "left-start" "left-end"}} @default="bottom">
+        Placement for the preferred starting position of the popover relative to the toggle element.
+        <br />
+        <br />
+        _Notice: if `@enableCollision` is set, the popover will automatically shift position to remain visible when near the edges of the screen regardless of the starting placement._
+      </C.Property>
+      <C.Property @name="anchoredPositionOptions.strategy" @type="enum" @values={{array "absolute" "fixed"}} @default="absolute">
+        Controls how the layout of the popover is applied using the CSS `position` property. Since the component uses the [native web Popover API](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API) which promotes the popover to the [top layer](https://developer.mozilla.org/en-US/docs/Glossary/Top_layer), there is no need to use the `fixed` position to avoid stacking context conflicts, but we leave the option open for edge cases in which this is needed anyway.
+        <br />
+        <br />
+        _Notice: if the position/strategy is set to `fixed`, the rendering of the popover becomes jagged when the page is scrolled._
+      </C.Property>
+      <C.Property @name="anchoredPositionOptions.offsetOptions" @type="number|object" @default="0">
+        A number that represents the “distance” between the popover element and the toggle element. An object can also be passed, which enables to individually configure different axis.
+        <br />
+        <br />
+        For details see: [Floating UI > Offset > Options](https://floating-ui.com/docs/offset#options).
+        <br />
+        <br />
+        _Notice: this options can be used to control the relative position of the arrow in relation to the toggle._
+      </C.Property>
+      <C.Property @name="anchoredPositionOptions.enableCollisionDetection" @type="boolean|string" @values={{array "true" "false" "flip" "shift" "auto"}} @default="false">
+        This property controls if the popover should automatically adapt its position to remain visible when near the edges of the screen. It can be enabled for both axes setting it to `true`, or a single axes can be chosen passing the `flip` or `shift` value. If set to `auto` it will automatically place the popover in the position where there's more space available, but in this case it will ignore the `placement` value.
```

Comment:
```suggestion
        This property controls if the popover should automatically adapt its position to remain visible when near the edges of the screen. It can be enabled for both axes by setting it to `true`, or a single axes can be chosen by passing either the `flip` or `shift` values. If set to `auto`, it will automatically place the popover in the position where there's more space available, but in this case it will ignore the `placement` value.
```